modules/tinymce/src/core/main/ts/api/util/XHR.ts

```diff
@@ -59,10 +59,152 @@ const XHR: XHR = {
+   * </tr>
+   * <tr>
+   * <td>
+   * <p><code>async<br /></code></p>
+   * </td>
+   * <td>Optional</td>
+   * <td>
+   * <p><code>boolean</code></p>
+   * </td>
+   * <td>When <code>false</code>, the request will be synchronous. Set to <code>true</code> by default.</td>
+   * </tr>
+   * <tr>
+   * <td>
+   * <p><code>content_type<br /></code></p>
+   * </td>
+   * <td>Optional</td>
+   * <td>
+   * <p><code>string</code></p>
+   * </td>
+   * <td>Used to define the mime-type of the data. Similar to
+   * <a href="https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/overrideMimeType">
+   * the XMLHttpRequest.overrideMimeType method</a>.</td>
+   * </tr>
+   * <tr>
+   * <td>
+   * <p><code>crossDomain<br /></code></p>
+   * </td>
+   * <td>Optional</td>
+   * <td>
+   * <p><code>boolean</code></p>
+   * </td>
+   * <td>When <code>true</code>, the withCredentials property will be set to <code>true</code>.
+   * For information on the withCredentials property, see:
+   * <a href="https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/withCredentials">
+   * MDN Web Docs - XMLHttpRequest.withCredentials</a>.</td>
+   * </tr>
+   * <tr>
+   * <td>
+   * <p><code>data<br /></code></p>
+   * </td>
+   * <td>Optional</td>
+   * <td>
+   * <p><code>any</code></p>
+   * </td>
+   * <td>The data to be sent to the specified URL (<code>url</code>). For information on valid data inputs,
+   *  see: <a href="https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/send#Syntax">
+   *  MDN Web Docs - XMLHttpRequest.send() Syntax</a></td>
+   * </tr>
+   * <tr>
+   * <td>
+   * <p><code>requestheaders</code></p>
+   * </td>
+   * <td>Optional</td>
+   * <td>
+   * <p><code>Record&lt;string, { key: string; value: string }&gt;</code></p>
+   * </td>
+   * <td>Allows for the definition of additional
+   *  <a href="https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/setRequestHeader">
+   *  header-values for the request header</a>.</td>
+   * </tr>
+   * <tr>
+   * <td>
+   * <p><code>type</code></p>
+   * </td>
+   * <td>Optional</td>
+   * <td>
+   * <p><code>string</code></p>
+   * </td>
+   * <td>
+   * <p>The <a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods">HTTP request method</a>,
+   *  such as: <code>'GET'</code>, <code>'POST'</code>, <code>'PUT'</code> and <code>'DELETE'</code>.</p>
+   * <p>By default:</p>
+   * <ul>
+   * <li>If no data is provided with the request, the <code>'GET'</code> method will be used.</li>
+   * <li>If data is provided with the request, the <code>'POST'</code> method will be used.</li>
+   * </ul>
+   * </td>
+   * </tr>
+   * <tr>
+   * <td>
+   * <p><code>error_scope</code></p>
+   * </td>
+   * <td>Optional</td>
+   * <td>
+   * <p><code>object</code></p>
+   * </td>
+   * <td>Sets the <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/this">
+   * '<code>this</code>'</a> operator of the error callback function.</td>
```

Comment:
Still referring to `operator` here.

Reply:
:hankey:

Reply:
Fixed by https://github.com/tinymce/tinymce/pull/5924/commits/24059ba1645d967d6052ea2ddefc46dcf66f8310